src/schema/schema.graphql

```diff
@@ -1804,6 +1815,8 @@ type Query {
 		folders: String,
 		includeGal: Boolean
 	): AutoCompleteResponse
+	downloadMessage(id: ID!): SMimeMessage
+	folder(id: ID!): Folder
```

Comment:
is this needed?

Reply:
no.. merge issue i guess .. removing folder ..